packages/core/test/actor.test.ts

```diff
@@ -805,89 +805,6 @@ describe('actors', () => {
     expect(initialState.context.ref!.send).toBeDefined();
   });
 
-  describe('autoForward option', () => {
-    const pongActorMachine = createMachine({
-      id: 'server',
-      initial: 'waitPing',
-      states: {
-        waitPing: {
-          on: {
-            PING: 'sendPong'
-          }
-        },
-        sendPong: {
-          entry: [sendParent('PONG'), raise('SUCCESS')],
-          on: {
-            SUCCESS: 'waitPing'
-          }
-        }
-      }
-    });
-
-    it('should not forward events to a spawned actor by default', () => {
-      let pongCounter = 0;
-
-      const machine = createMachine<any>({
-        id: 'client',
-        context: { counter: 0, serverRef: undefined },
-        initial: 'initial',
-        states: {
-          initial: {
-            entry: assign((_, __, { spawn }) => ({
-              serverRef: spawn(pongActorMachine)
-            })),
-            on: {
-              PONG: {
-                actions: () => ++pongCounter
-              }
-            }
-          }
-        }
-      });
-      const service = interpret(machine);
-      service.start();
-      service.send('PING');
-      service.send('PING');
-      expect(pongCounter).toEqual(0);
-    });
-
-    it('should not forward events to a spawned actor when { autoForward: false }', () => {
-      let pongCounter = 0;
-
-      const machine = createMachine<{
-        counter: number;
-        serverRef?: ActorRef<any>;
-      }>({
-        id: 'client',
-        context: { counter: 0, serverRef: undefined },
-        initial: 'initial',
-        states: {
-          initial: {
-            entry: assign((ctx, _, { spawn }) => ({
-              ...ctx,
-              serverRef: spawn(
-                pongActorMachine
-                // {
-                //   autoForward: false
-                // }
-              )
```

Comment:
The target branch already removed support for `spawn`+`autoForward` - hence this whole test block is redundant